nautobot/dcim/choices.py

```diff
@@ -909,6 +909,31 @@ class InterfaceModeChoices(ChoiceSet):
     )
 
 
+class InterfaceStatusChoices(ChoiceSet):
+
+    STATUS_PLANNED = "planned"
+    STATUS_ACTIVE = "active"
+    STATUS_DECOMMISSIONING = "decommissioning"
+    STATUS_FAILED = "failed"
+    STATUS_MAINTENANCE = "maintenance"
+
+    CHOICES = (
+        (STATUS_PLANNED, "Planned"),
+        (STATUS_FAILED, "Failed"),
+        (STATUS_ACTIVE, "Active"),
+        (STATUS_DECOMMISSIONING, "Decommissioning"),
+        (STATUS_MAINTENANCE, "Maintenance"),
+    )
+
+    CSS_CLASSES = {
```

Comment:
Is CSS_CLASSES actually used/needed anywhere? If not I'd say let's not add it.

Reply:
Its been used by [export_statuses_from_choiceset()](https://github.com/nautobot/nautobot/blob/timizuo-GH984-interface-status/nautobot/extras/management/__init__.py#L80), [2bbef5af517R15](https://github.com/nautobot/nautobot/pull/1753/files#diff-0d98c24df83f4a421cf3962f7a1bc886836279d0b9879439d4a8b2bbef5af517R15)